app/collections/collaborators.jsx

```diff
@@ -0,0 +1,375 @@
+    });
+  }
+
+  render() {
+    const { owner } = this.state;
+
+    return (
+      <p>
+        {owner && (<strong>{owner.login}</strong>)}
+
+        { owner.display_name !== this.props.collectionOwner && (
+          <button type="button" className="pill-button" onClick={this.removeRoles}>Remove</button>
+        )}
+
+        <span className="columns-container inline">
+          {POSSIBLE_ROLES.map((role) => {
+            const checkedRole = this.props.roleSet.roles.includes(role);
+            const boldRole = checkedRole ? <strong>{ROLES_INFO[role].label}</strong> : ROLES_INFO[role].label;
+            const disableCheck = role === 'owner' || this.state.saving || owner.display_name === this.props.collectionOwner;
+
+            return (
+              <label htmlFor="role" key={role}>
+                <input type="checkbox" name={role} checked={checkedRole} disabled={disableCheck} onChange={this.toggleRole.bind(this, role)} />{' '}
+                {boldRole}
+              </label>
+            );
+          })}
+        </span>
+      </p>
+    );
+  }
+}
+
+RoleRow.propTypes = {
+  collectionOwner: React.PropTypes.string,
+  onRemove: React.PropTypes.func,
+  roleSet: React.PropTypes.shape({
+    delete: React.PropTypes.func,
+    get: React.PropTypes.func,
+    roles: React.PropTypes.array,
+    update: React.PropTypes.func
+  })
+};
+
+export class CollectionCollaborators extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      error: null,
+      hasSettingsRole: false,
+      owner: null,
+      roleSets: []
+    };
+
+    this.update = this.update.bind(this);
+    this.handleCollaboratorChange = this.handleCollaboratorChange.bind(this);
+  }
+
+  componentDidMount() {
+    checkIfCollectionOwner(this.props.user, this.props.collection)
+    .then((hasSettingsRole) => {
+      this.setState({ hasSettingsRole });
+    });
+    this.update();
+  }
+
+  update(callback = () => {}) {
+    const promise = Promise.all([
+      apiClient.type('collection_roles').get({ collection_id: this.props.collection.id }),
+      this.props.collection.get('owner')
+    ]);
+
+    promise.then(([roleSets, owner]) => {
+      this.setState({ roleSets, owner }, callback);
+    });
+  }
+
+  handleCollaboratorChange() {
+    this.props.collection.uncacheLink('collection_roles');
+    this.update();
+  }
+
+  ownerRoleFirst() {
+    const { roleSets } = this.state;
+
+    roleSets.map((set, i) => {
+      if (set.roles.includes('owner')) {
+        const test = roleSets.splice(i, 1);
+        roleSets.unshift(test[0]);
+      }
+    });
+    return roleSets;
+  }
+
+  render() {
+    const { owner } = this.state;
+    const roleSets = this.ownerRoleFirst();
+
+    if (this.state.hasSettingsRole) {
+      return (
+        <div className="collection-settings-tab">
+          {this.state.error && (<p className="form-help error">{this.state.error.toString()}</p>)}
+
+          {owner && (
+            <div>
```

Comment:
It seems like this should be an unordered list with the `RoleRow` as list items.